charts/seed-bootstrap/charts/nginx-ingress/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: nginx-ingress
+    chart: nginx-ingress-0.8.0
+    heritage: Tiller
+    release: addons
```

Comment:
These labels can be removed from all resources inside this chart

Reply:
Can I remove those three labels from all the charts? They seem to be obsolete.

Reply:
Generally yes, but the charts will be replaced as part of #2754 anyways, so no need to invest into it now.